pkg/operator/controller/ingress/dns.go

```diff
@@ -56,10 +56,10 @@ func (r *reconciler) ensureWildcardDNSRecord(ic *operatorv1.IngressController, s
 		}
 	}
 
-	return true, current, nil
+	return haveWC, current, nil
 }
 
-// ensureWildcardDNSRecord will return any necessary wildcard DNS records for the
+// desiredWildcardDNSRecord will return any necessary wildcard DNS records for the
```

Comment:
`desiredWildcardDNSRecord` → `desiredWildcardRecord`.

Reply:
Whoops. This typo brings up an inconsistency, however.
<bikeshed>
Should `desiredWildCardRecord` actually be called `desiredWildCardDNSRecord` since the other functions in `dns.go` follow that pattern?
</bikeshed>

Reply:
Ah, good point.  I wouldn't object to renaming `desiredWildcardRecord` to `desiredWildcardDNSRecord`.  Bonus: That would save you the trouble of changing the godoc again!

Reply:
fixed! :smile: